### PR TITLE
static check

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ secrets.GO_VERSION }}
-    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck ./...
+    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck -exclude errcheck_excludes.txt ./...
```
Comment (leecalcote) [inline .github/workflows/ci.yml]: @ashiscs  has the `errcheck_excludes.txt` been defined somewhere?

**Comment by ashiscs** [comment]
@leecalcote sir, this pr was landed by @gunjanmimo 
I also went through it but errcheck_excludes.txt is not defined anywhere!!

**Comment by leecalcote** [comment]
@ashiscs, oh, oops. I mistakenly thought that you had created this PR. I see now that @gunjanmimo did. :)

**Comment by leecalcote** [comment]
@gunjanmimo given the latest PR and update to this workflow, I think we're ok to close this PR. What do you think?

**Comment by leecalcote** [comment]
@nveenjain good to close out, I think.

**Comment by kushthedude** [comment]
Consul actions are still failing I will make a PR to resolve them.

**Comment by ashiscs** [comment]
@kushthedude consul actions are not failing and was fixed by this pr https://github.com/layer5io/meshery-consul/pull/38

**Pull request closed without merging**